models/social_connection.py

```diff
@@ -0,0 +1,69 @@
+import json
```

Comment:
Do you think it would be better to do as dicts in each model? I thought we were trying to move away from those JSON properties because it made more complex queries and mutations more difficult. I'm open to changing it if we think it's better done by a different approach, though